<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="ROZARIO-PC" ignored="0" name="EmployeesRestAPITest" tests="5" failures="1" timestamp="22 Jul 2019 05:10:37 GMT" time="24.3" errors="0">
  <testcase name="verifyResponse" time="0.138" classname="com.employeeproject.testcases.TC001_Get_All_Employee_Data"/>
  <testcase name="verifyResponsebody" time="0.003" classname="com.employeeproject.testcases.TC002_Single_Employee_Data"/>
  <testcase name="verifyResponseBody" time="0.003" classname="com.employeeproject.testcases.TC003_Create_NewEmployee_record">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.employeeproject.testcases.TC003_Create_NewEmployee_record.verifyResponseBody(TC003_Create_NewEmployee_record.java:58)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- verifyResponseBody -->
  <testcase name="validateResponseBody" time="0.004" classname="com.employeeproject.testcases.TC004_Update_Employee_record"/>
  <testcase name="validateResponseBody" time="0.002" classname="com.employeeproject.testcases.TC005_Delete_Employee_record"/>
</testsuite> <!-- EmployeesRestAPITest -->
